src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/Domain/HistoricalPayment.cs

```diff
@@ -0,0 +1,37 @@
+﻿using SFA.DAS.Payments.DCFS.Domain;
+using SFA.DAS.ProviderPayments.Calc.Refunds.Infrastructure.Entities;
+using SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities;
+
+namespace SFA.DAS.ProviderPayments.Calc.Refunds.Domain
+{
+    public class HistoricalPayment : PaymentEntity
```

Comment:
Is `PaymentEntity` an entity or a domain model? if an entity then this should be changed as we don't want a domain model inheriting from a data entity (seems wrong to me anyway). if its a domain model then it should be renamed.

Reply:
This code is taken from the 'Make a payment' branch (the one we didn't get time to review.) It's far from ideal that we have a domain object having a dependency on an Entity (however we have done this with the refund domain object).  Please discuss with Carl and see his view